test/controllers/request_history_controller_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Imposter.RequestHistoryControllerTest do
+	use Imposter.ConnCase
+
+	alias Imposter.RequestHistory
+
+	@attrs %{method: "GET", path: "/baz", response: %{foo: "this is a get"}}
+
+	setup do
+		conn = conn() |> put_req_header("accept", "application/json")
+		{:ok, conn: conn}
+	end
+
+	test "lists all entries on index", %{conn: conn} do
```

Comment:
have a version of this test that returns some actual entries too, and for RoutesController too.